src/+mpecopt/Solver.m

```diff
@@ -1157,5 +1334,454 @@ function create_lpec_functions(obj)
 
             obj.lpec_casadi = lpec_casadi;
         end
+
+        function [solution,stats] = phase_II(obj,mpec_casadi,lpec_casadi,dims,opts,solver_initialization,stats,phase_ii)
```

Comment:
would make sense to have also phase_I as a method to be more consistent.
can be a seperate PR/issue for requesting refactoring.